index.js

```diff
@@ -1,55 +1,38 @@
 "use strict";
 
 module.exports = {
-  extends: "eslint:recommended",
-  globals: {
-    DataView: false,
-    ArrayBuffer: false,
-    Float32Array: false,
-    Float64Array: false,
-    Int16Array: false,
-    Int32Array: false,
-    Int8Array: false,
-    Uint16Array: false,
-    Uint32Array: false,
-    Uint8Array: false,
-    Uint8ClampedArray: false,
+  extends: ["eslint:recommended", "prettier"],
```

Comment:
Are we sure this works? I removed prettier from my own config that extends cesium/node and suddenly got a ton of "Strings myst use singleqoute" type errors, so I had to put back:

```
  "extends": [ "cesium/node", "prettier"],
```

Same for `"cesium/browser". Did you try this change in CesiumJS itself and were you able to remove direct extending of prettier there?

Reply:
I did test it earlier, and I just confirmed with the latest version of this repo (using `npm link`) - I used the [`eslint-test`](https://github.com/CesiumGS/cesium/tree/eslint-test) branch of CesiumJS].

Reply:
I tested as follows:
```
git clean -xdf
npm link eslint-config-cesium
npm i
npx eslint --print-config server.cjs # The output of this command should include the rules from the prettier config
```

Reply:
Thanks, I think my understanding here was off and the print-config command (which I was unaware) is super useful. 

Can you confirm this is correct?

1. We used to have the double quote rule turned on.
2. We removed it because the prettier config handles it.
3. The rule is actually off now, so it no longer will show an eslint error, but that's because prettier automatically fixes it so the rule itself is unneeded.

Is that correct?